src/components/ReviewCreateForm/InformCancelModal.tsx

```diff
@@ -0,0 +1,39 @@
+import ModalPortal from '@/components/Modal/ModalPortal';
+import { useNavigate } from 'react-router-dom';
+
+type CancelModal = {
```

Comment:
```suggestion
type CancelModalPropsType = {
```
p3;
props 타입 뒤에 propsType 넣어주세요.

Reply:
오 propsType 까지 붙이나여?? props으로 넘기는 타입에 propsType 붙이는 거 맞나여??